smartcontractkit/external-adapters-js
# enable turning on TLS for Ada Balance adapter

## Closes 26233

## Description

A node operator is asking that we enable a TLS connection to connect the ADA balance adapter with the Cardano Ogmios node.  This PR adds support for this.

## Changes

- Enable TLS support for connecting to the Cardano Ogmios node
- Upgrade the Ogmios client library

## Testing

 - Run adapter without `IS_TLS_ENABLED` turned on and ensure it is still responding correctly
 - Run adapter with `IS_TLS_ENABLED` turned on and ensure it is still responding correctly

Message me directly for the environment variables to run the adapter with

## Quality Assurance
